### PR TITLE
Add Community Choice 2021: Results newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
✅ 

Big newspost, links are script generated so if one's wrong they'll probably all be wrong somewhere.

### CODE REVIEW

File: news/2022-02-10-community-choice-results.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+- [**Risshuu feat. Choko - Take**](https://osu.ppy.sh/beatmapsets/1377628) by [**LNP-**](https://osu.ppy.sh/users/4402842) with **211** votes.
+- [**Camellia - OOPARTS**](https://osu.ppy.sh/beatmapsets/1505323) by [**-mint-**](https://osu.ppy.sh/users/8976576) with **202** votes.
+- [**t+pazolite - Tempestissimo**](https://osu.ppy.sh/beatmapsets/1179815) by [**Kuo Kyoka**](https://osu.ppy.sh/users/2596306) with **183** votes.
+- [**BlackY - AlphaOmega (Evolutionary Extended ver.)**](https://osu.ppy.sh/beatmapsets/1329899) by [**Shima Rin**](https://osu.ppy.sh/users/6089608) with **176** votes.
+- [**Camellia - Flamewall**](https://osu.ppy.sh/beatmapsets/1270879) by [**riunosk**](https://osu.ppy.sh/users/5594381) with **166** votes.
+- [**Zekk - Re_Construct**](https://osu.ppy.sh/beatmapsets/1492528) by [**Lita-**](https://osu.ppy.sh/users/15205070) with **160** votes.
+- [**MisoilePunch - VVelcome!!**](https://osu.ppy.sh/beatmapsets/1235288) by [**Aruel**](https://osu.ppy.sh/users/3984370) with **155** votes.
+- [**MisoilePunch ~Takenoko Soe~ - Staring at star**](https://osu.ppy.sh/beatmapsets/920688) by [**Murumoo**](https://osu.ppy.sh/users/8001433) with **154** votes.
```
Comment (mousesalt) [inline news/2022-02-10-community-choice-results.md:158]:
```suggestion
- [**MisoilePunch \~Takenoko Soe\~ - Staring at star**](https://osu.ppy.sh/beatmapsets/920688) by [**Murumoo**](https://osu.ppy.sh/users/8001433) with **154** votes.
```

**Review by mousesalt — CHANGES_REQUESTED** [review]

File: news/2022-02-10-community-choice-results.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+- [**NIWASHI - Playing with Ruby**](https://osu.ppy.sh/beatmapsets/1328323) by [**Cychloryn**](https://osu.ppy.sh/users/6921736) with **15** votes.
+- [**ASA-CHANG & Junray - Kage no Nai Hito**](https://osu.ppy.sh/beatmapsets/1519045) by [**Zetera**](https://osu.ppy.sh/users/587737) with **14** votes.
+- [**METAROOM - SPIKEY BOI**](https://osu.ppy.sh/beatmapsets/1424830) by [**Nifty**](https://osu.ppy.sh/users/4956097) with **14** votes.
+- [**glaive - astrid**](https://osu.ppy.sh/beatmapsets/1371469) by [**radar**](https://osu.ppy.sh/users/7131099) with **13** votes.
+
+## osu!catch
+
+[![](/wiki/shared/news/2022-02-10-community-choice-results/take.jpg)](https://osu.ppy.sh/beatmapsets/1295206)
+
+Screaming to the top of the voting at a whopping 5 to 315+ bamboo per minute is none other than [**Risshuu feat. Choko - Take**](https://osu.ppy.sh/beatmapsets/1295206) by [**Kukkai**](https://osu.ppy.sh/users/7811952) with **72** total votes as the newest Community Favourite map. Good luck getting the track out of your head!
```
Comment (Walavouchey) [inline news/2022-02-10-community-choice-results.md:98]: style suggestion, also overlooking the missing plural s

```suggestion
Screaming to the top of the voting at a whopping 5 to 315+ BPM (Bamboo Per Minute) is none other than [**Risshuu feat. Choko - Take**](https://osu.ppy.sh/beatmapsets/1295206) by [**Kukkai**](https://osu.ppy.sh/users/7811952) with **72** total votes as the newest Community Favourite map. Good luck getting the track out of your head!
```

File: news/2022-02-10-community-choice-results.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+Lavished with praise, criticism, despair and jubilation alike, osu!mania's newest Community Favourite is none other than [**Chroma - Dark Sheep**](https://osu.ppy.sh/beatmapsets/939798) by [**Feylos**](https://osu.ppy.sh/users/4565040) with **278** votes.
+
+Here's a brief snippet from the comments of the map:
+
+- "The feel when osu!mania finally goes moderately distant from the anime monopoly"
+- "You don't care about it's overwhelming difficulty, and actually nominate it. You broke the rules. We're very thankful for that."
+- "the song from rowbeets!!1!!1111! 😱😱😱😱😱"
+
+Blood, sweat and tears are the polish that makes a map truly shine, and it is clear that Dark Sheep has them in abundance if the community is anything to go by. Well done!
```
Comment (Walavouchey) [inline news/2022-02-10-community-choice-results.md:146]:
```suggestion
Blood, sweat and tears are the polish that makes a map truly shine, and it is clear that *Dark Sheep* has them in abundance if the community is anything to go by. Well done!
```

File: news/2022-02-10-community-choice-results.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+- [**MisoilePunch - VVelcome!!**](https://osu.ppy.sh/beatmapsets/1235288) by [**Aruel**](https://osu.ppy.sh/users/3984370) with **155** votes.
+- [**MisoilePunch \~Takenoko Soe\~ - Staring at star**](https://osu.ppy.sh/beatmapsets/920688) by [**Murumoo**](https://osu.ppy.sh/users/8001433) with **154** votes.
+
+[![](/wiki/shared/news/2022-02-10-community-choice-results/caravan.jpg)](https://osu.ppy.sh/beatmapsets/1086739)
+
+A trek over to the mapping community's side of things has us set off on a journey with [**John Wasson - Caravan**](https://osu.ppy.sh/beatmapsets/1086739) by [**Evening**](https://osu.ppy.sh/users/2193881) with **24** votes, barely overtaking its nearest rival by a mere four votes.
+
+A nine minute journey into 8/9 smooth jazz by one of osu!mania's most reputable mappers is possibly the strongest indictment yet for just how good of a map it must be. Getting anybody to play nine minutes of *anything* is a testament to the experience, let alone the herculean effort involved to render this critical masterpiece mappable to begin with.
```
Comment (Walavouchey) [inline news/2022-02-10-community-choice-results.md:164]:
```suggestion
A nine-minute journey into 8/9 smooth jazz by one of osu!mania's most reputable mappers is possibly the strongest indictment yet for just how good of a map it must be. Getting anybody to play nine minutes of *anything* is a testament to the experience, let alone the herculean effort involved to render this critical masterpiece mappable to begin with.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2022-02-10-community-choice-results.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+- [**POLKADOT STINGRAY - Otoshimae**](https://osu.ppy.sh/beatmapsets/1301330) by [**moph**](https://osu.ppy.sh/users/2233878) with **37** votes.
+- [**UKRampage - Jack-the-Ripper**](https://osu.ppy.sh/beatmapsets/1333932) by [**DeviousPanda**](https://osu.ppy.sh/users/4966334) with **35** votes.
+
+## osu!taiko
+
+[![](/wiki/shared/news/2022-02-10-community-choice-results/drip.jpg)](https://osu.ppy.sh/beatmapsets/1421463)
+
+Living up 2021 to the fullest, the osu!taiko community ejected all reason out of the airlock and threw their weight behind [**Leonz - Among Us Trap Remix (Among Drip Theme Song)**](https://osu.ppy.sh/beatmapsets/1421463) with **78** votes.
+
+There's very little to say about this map other than one thing: ඞ. He does have drip though.
```
[the remark this answers is not in the document]
Comment (Walavouchey) [reply news/2022-02-10-community-choice-results.md:63]: good sir in your words there is very little to say about this

**Review by peppy — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**